flows/default.js

```diff
@@ -300,9 +300,10 @@ const loadFlow = (app) => {
       <#CBX54ACPJ> - Share your photos!
       <#CC78UKWAC> - :jenga_sleep:
       <#C8P6DHA3W> - Don't enter if you're hungry!
-      <#C010SJJH1PT> - Learn about cooking!
+      <#C0EA9S0A0> - For general coding! There's also <#C0131FX5K98>, <#C016G8A6MKJ>, <#C012LPZUAPR>, <#C019NDLMJCV>, <#C0120KJH1SB>, <#C0121LVV79P>, and <#CLWA83V7Y>, to name a few.
```

Comment:
maybe we should keep the list format, tho I'll leave that up to Matthew

Reply:
Lol K